server/service/individualServices/ReadInventoryData.js

```diff
@@ -265,18 +310,504 @@ async function formulateEquipmentInfo(equipmentInfoList) {
-    } else if (category === EQUIPMENT.EQUIPMENT_CATEGORY.MODULE + ":" + EQUIPMENT.EQUIPMENT_CATEGORY.FULL_OUTDOOR_UNIT) {
+    } else if (category === EQUIPMENT.MODULE + EQUIPMENT.EQUIPMENT_CATEGORY.FULL_OUTDOOR_UNIT) {
       equipmentInfo.device = equipment;
     }
   }
   return equipmentInfo;
-}
\ No newline at end of file
+}
+
+/**
+* Function to gather the Configured group of resources (of type AirInterface or WireInterface) that are transporting fragments of the same Ethernet frames
+* If the client ethernet-container is shared with an air-interface(radio-link-bonding), RequestForProvidingAcceptanceDataCausesReadingConfigurationFromCache shall be used to get link-id
+* Else if the client ethernet-container is shared with the wire-interface (physical-link-aggregation), RequestForProvidingAcceptanceDataCausesAnalysingTheAggregation shall be used to get original-ltp-name
+* @param {String} mountName Identifier of the device at the Controller
+* @param {Object} ltpStructure ControlConstruct provided from cache
+* @param {String} uuidUnderTest Identifier of the air-interface under test
+* @param {Object} requestHeaders Holds information of the requestHeaders like Xcorrelator , CustomerJourney,User etc.
+* @param {Integer} traceIndicatorIncrementer traceIndicatorIncrementer to increment the trace indicator
+* @returns {Object} return values of installed-firmware list and traceIndicatorIncrementer
+*/
+async function FetchConfiguredGroupOfAirInterfaces(mountName, ltpStructure, uuidUnderTest, requestHeaders, traceIndicatorIncrementer) {
+  let configuredGroupOfAirInterfacesResponse = {};
+  let configuredGroupOfAirInterfaceList = [];
+  try {
+    /****************************************************************************************
+     * extracting list of serving physic ltps for air-interface uuidUnderTest
+     ****************************************************************************************/
+    let firstLayerToReachEthernetContainer = [PURE_ETHERNET_STRUCTURE.MODULE + PURE_ETHERNET_STRUCTURE.LAYER_PROTOCOL_NAME, HYBRID_MW_STRUCTURE.MODULE + HYBRID_MW_STRUCTURE.LAYER_PROTOCOL_NAME];
+    let secondLayerToReachEthernetContainer = [ETHERNET_CONTAINER.MODULE + ETHERNET_CONTAINER.LAYER_PROTOCOL_NAME];
+    let interfaceListToReachEthernetContainer = [firstLayerToReachEthernetContainer, secondLayerToReachEthernetContainer];
+    let airInterfaceLtpUnderTest = await LtpStructureUtility.getLtpForUuidFromLtpStructure(uuidUnderTest, ltpStructure);
+    let clientContainerLtp = await LtpStructureUtility.getHierarchicalClientLtpForInterfaceListFromLtpStructure(airInterfaceLtpUnderTest, interfaceListToReachEthernetContainer, ltpStructure);
+    if (Object.keys(clientContainerLtp).length > 0) {
+      let servingPhysicLtpList = await getServingPhysicLtpList(clientContainerLtp, ltpStructure);
+      if (servingPhysicLtpList != undefined || servingPhysicLtpList.length != 0) {
+        for (let i = 0; i < servingPhysicLtpList.length; i++) {
```

Comment:
From this flow, I understood that even if there is no radio-link-bonding available , there will be one link-id available in the configured group of air-interface.
Can you please include a check to identify if there is a radio-link-bonding and If there is a bonding , then list the group of connected interfaces.